### PR TITLE
Copy updates for German sept landing page (#9361)

### PR DESCRIPTION
**Pull request by alexgibson**
## Description
A couple of minor copy amends for German

## Issue / Bugzilla link
#9361

### CODE REVIEW

File: bedrock/firefox/templates/firefox/campaign/unfck/index.de.html [hunk 1]
```diff
@@ -20,7 +20,7 @@
 
 {% set referrals = '?utm_source=www.mozilla.org&utm_medium=referral&utm_campaign=firefox-unfck' %}
 
-{% set tw_text = 'Schütze das Web das du liebst. Unfck the rest.'|urlencode %}
+{% set tw_text = 'Schütze das Web, das du liebst. Unfck the rest.”'|urlencode %}
```
Comment (stephaniehobson) [inline bedrock/firefox/templates/firefox/campaign/unfck/index.de.html:23]: I don't see the matching open quote of this close quote. 🍝 ?

**Review by stephaniehobson — CHANGES_REQUESTED** [review]
Copy-pasta or some German punctuation I'm not famillir with...

**Comment by alexgibson** [comment]
@stephaniehobson thanks updated - I've added in a couple more minor updates since this PR was opened.

File: bedrock/firefox/templates/firefox/campaign/unfck/index.de.html [hunk 2]
```diff
@@ -131,31 +139,31 @@ <h2 class="c-section-title">Die Unfck-Checkliste:</h2>
...
 
       {% call share_item(
         title='Netflix & change',
-        info='Schau dir die Dokumentation <em>“The Social Dilemma”</em> mit unseren Freunden des Center for Human Tech an und erfahre mehr darüber, wie Milliarden von Menschen zur Handelsware der Konzerne werden.'|safe,
-        link_text='Zur Doku',
-        link_url='https://blog.mozilla.org/firefox/de/Das-Dilemma-mit-den-sozialen-Medien/',
-        id='social-dilemma',
+        info='Schau dir die Dokumentation <em>“The Social Dilemma”</em> von Exposure Labs und dem Center for Humane Tech an und erfahre mehr darüber, wie Milliarden von Menschen zur Handelsware der Konzerne werden.'|safe,
+        link_text='Mehr zur Doku',
+        link_url='https://blog.mozilla.org/firefox/de/soziale-medien-warum-kostenlos-doch-seinen-preis-hat/',
```
Comment (amychurchwell) [inline bedrock/firefox/templates/firefox/campaign/unfck/index.de.html:151]: this URL is not found, is it correct?

Comment (alexgibson) [reply bedrock/firefox/templates/firefox/campaign/unfck/index.de.html:151]: yes it will be live by tomorrow, thanks

**Review by amychurchwell — APPROVED** [review]
lgtm r+

**Pull request merged**